src/featuremap/tests/system/test_create_somatic_pileup_featuremap.py

```diff
@@ -61,3 +61,41 @@ def test_create_somatic_pileup_featuremap(tmp_path, resources_dir):
     assert num_variants == expected_num_variants
     # assert a single entry per variant
     assert all(cons["count"] == 1 for cons in cons_dict.values())
+
+
+def test_parse_samtools_mpileup(tmp_path, resources_dir):
```

Comment:
I think this should be in a different file. 
it is a system test for ref_nonref_per_base_window script.